src/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Leaderboard</title>
+    <link href="https://fonts.googleapis.com/css2?family=Salsa&display=swap" rel="stylesheet">
+  </head>
+  <body>
+    <main class="all">
+      <div class="mainarea">
+        <h1 class="maintitle">Leaderboard</h1>
+      </div>
+      <div class="leaderboard">
+        <h3 class="scores">Recent scores</h3>
+        <button class="refreshBtn">Refresh</button>
+      </div>
+      <div class="leaderboardscores">
+          <div class="score">Name:100</div>
+          <div class="score">Name:69</div>
+          <div class="score">Name:107</div>
+          <div class="score">Name:93</div>
+          <div class="score">Name:40</div>
+          <div class="score">Name:230</div>
+          <div class="score">Name:145</div>
+      </div>
```

Comment:
- [ ] It would be great to use semantic HTML tags for this feature. Since this is a "List" of scores, the `<ul>` tag describes best what you want to structure, so please consider applying this change! 🤓 🔧 

```suggestion
      <ul class="leaderboardscores">
          <li class="score">Name:100</li>
          <li class="score">Name:69</li>
          <li class="score">Name:107</li>
          <li class="score">Name:93</li>
          <li class="score">Name:40</li>
          <li class="score">Name:230</li>
          <li class="score">Name:145</li>
      </ul>
```